app/js/components/drugOrderEntry/addForm/AddForm.jsx

```diff
@@ -81,6 +83,16 @@ export class AddForm extends React.Component {
     this.setState({ formType: formType.trim() });
   }
 
+  checkIfDrugHasActiveOrder = (drug) => {
+    const { filteredOrders } = this.props.fetchOrdersReducer;
```

Comment:
In `componentDidMount`, `fetchOrders` action returns only 10 orders, which is the default limit meaning your filtered orders are a proportion of all relevant orders in the database. If you are going to look through these 10 orders to determine if a drug name exists then I think this fails in case the drug was not returned among the 10 orders. Does this make sense to you? I would rather you find a way to check all orders rather than those 10 returned when you mount the component.

Reply:
Please use destructuring props assignment.

Reply:
Will check on this. I had missed the limit parameter. I can add a function to ferchOrders.js action file to fetch all orders from the API.

Reply:
Instead of getting all orders everytime a form is mounted, I think there is already a backend route to check for the existence of an order.

Reply:
Done